src/features/peers/redux/middleware.ts

```diff
@@ -4,13 +4,16 @@ import { SocketService } from "services/socketService";
 import { Middleware } from "redux";
 import { AppState } from "store";
 
-type Dependencies = {
+export type PeersDependencies = {
```

Comment:
q: I see you're now exporting `PeersDependencies`, any reason for that?

Reply:
Previously I need it in my test, now I don't need to export it.
Will remove it